### PR TITLE
Latest JDK in Travis

### PR DESCRIPTION
**Pull request by jbescos**
image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Distribution License v. 1.0, which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+dist: focal
+
+sudo: required
```
Comment (pzygielo) [inline .travis.yml:12]: Why?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+language: java
+
+services:
+  - docker
```
Comment (pzygielo) [inline .travis.yml:17]: What is it used for?

Comment (jbescos) [reply .travis.yml]: java is needed to have some tools installed, like maven, JDK, etc.

Docker was a copy-paste. It seems it is not required so I have removed it.

File: .travis.yml (hunk 1 above)
Comment (jbescos) [reply .travis.yml]: I set it in the same way it was with mail:
https://github.com/eclipse-ee4j/mail/pull/560
There was a comment to use focal.

I have removed it, now it is using Xenial.

Comment (lukasj) [reply .travis.yml]: I believe the comment was not about xenial vs focal but about sudo

**Comment by jbescos** [comment]
> travisorg
> and on .com ee4j has no more credits to spend on running builds.

Should we discard this PR?.

File: .travis.yml (hunk 1 above)
Comment (jbescos) [reply .travis.yml]: I removed both (sudo and dist)

**Pull request closed without merging**